roles/rabbitmq/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Install rabbitmq-server rpm
+  yum:
+   name: rabbitmq-server
+   state: present
+
+- name: Start the rabbitmq service
+  service:
+   name: rabbitmq-server
+   state: started
+   enabled: yes
+
+- name: Added rabbitmq plugin
+  rabbitmq_plugin:
+   names: rabbitmq_management
+   state: enabled
+
+- name: Create rabbitmq vhost
+  rabbitmq_vhost:
+   name: '{{ sensu_rabbitmq_vhost }}'
+   state: present
+  notify: Restart rabbitmq
+
+# Add user to server and assign full access control on / vhost.
+# The user doesn't have permission rules for other vhosts
+- name: Configure rabbitmq permissions
```

Comment:
Same comment here; the sensu-specific configuration should probably live in the sensu role.